portfolio/src/main/webapp/script.js

```diff
@@ -1,27 +1,29 @@
 /**
   * This is the main javascript file for the portfolio page. Contains a showProjectDetail method.
+
+  Copyright 2019 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
   * @author: shradha-khapra
   */
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
 /**
   * Shows details of a specified project.
-  * @param {HTMLElement.itemId} buttonId The ID of the button that was clicked to invoke the function.
+  * @param {void} 
```

Comment:
This is not a void param. It is an event object. 

```suggestion
  * @param {!Event} event The event object received.
```
`!` indicates that the object can never be undefined or null.

https://developer.mozilla.org/en-US/docs/Web/API/Event